themes/SuiteP/css/editview.scss

```diff
@@ -5435,6 +5435,10 @@ table tr#delegates_search {
 /* popup in studio */
 .masked .yui-simple-dialog {
   // Overridding inline style
+  top: 50%!important;
```

Comment:
We don't use `!important` in `.scss` files, actualy I think nothing to do with the positioning on YUI dialogs in stylesheet, and simply enough to calling a `center()` method on Yahoo dialog, insert only one extra line in `InboundEmail.js` at line 271. After `show()`. 
https://github.com/salesagility/SuiteCRM/blob/hotfix-7.8.x/jssource/src_files/modules/InboundEmail/InboundEmail.js#L271
for e.g:
```
SI.listDlg.center();
```

Reply:
Ill just take a look into making these changes.

Thanks